busmall.js

```diff
@@ -48,7 +43,6 @@ function randomizer(e) {
   totalClicks += 1;
```

Comment:
Since this function does not actually randomize the images, you may want to give it a different name that better matches its purpose.